app/helpers/agent_helper.rb

```diff
@@ -82,6 +82,12 @@ def agent_type_icon(agent, agents)
     end
   end
 
+  def agent_type_select_options
+    Rails.cache.fetch('agent_type_select_options') do
+      Agent.types.map {|type| [agent_type_to_human(type.name), type, {title: h(Agent.build_for_type(type.name, User.new(id: 0), {}).html_description.lines.first.strip)}] }
```

Comment:
Can't we just make `html_description` a class method?  `description` is stored in a class, so you are not supposed to include anything specific to the current user in it anyway.